src/postinstall.js

```diff
@@ -55,9 +55,35 @@ async function warmGittarCache() {
   console.log('  Warm cache for project template.');
 
   try {
+    const maxTimeLimit = 7000;
```

Comment:
This timeout limit is debatable.

Reply:
We should set this based on how long it takes to succeed for users with poor internet. So I'd go much higher personally--maybe 15sec